Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/TodoConfiguration.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftLintCore
+
+enum TodoKeyword: String, CaseIterable {
+    case todo = "TODO"
+    case fixme = "FIXME"
+}
+
+struct TodoConfiguration: SeverityBasedRuleConfiguration, Equatable {
+    typealias Parent = TodoRule
+
+    @ConfigurationElement(key: "severity")
+    private(set) var severityConfiguration = SeverityConfiguration<Parent>(.warning)
+    @ConfigurationElement(key: "only")
+    private(set) var onlyKeywords: Set<String> = Set(TodoKeyword.allCases.map { $0.rawValue })
```

Comment:
Why not store the values as enums instead of transforming them to strings? That would make their usage more type-safe.

Reply:
Ah, that sounds great! Thanks.